### DISCUSSION TITLE
「开源之史」系列之二/七：自由软件的商业模式 —— Cygnus 在许可之外的探索 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#65] [Announcements]
# 「开源之史」系列之二/七：自由软件的商业模式 —— Cygnus 在许可之外的探索 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/05-01-emacs-and-cygnus-make-money/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

不是从海洋法系中获得灵感,
而是相反, 海洋法系的诞生就是从类似开源的社会现实中抽象出来的,
其根源很可能就是 海盗们的内部规约,

0: 先定个最基础的公约
1: 凡是没约定的都可以尝试
2: 尝试出问题时, 集体裁决出一个修订, 增补到公约中

凡事以上循环处置...

而 FLOSS/ 自动开源许可证, 只约定了一件事儿: 

> 不得再次封闭软件本身的开发可能空间;

这等于一次性拥有了所有的未来, 
当然比其它厂商先挖坑再等客户掉进来要合理的多...

问题不过是, 应该根据什么来收费;
闭源比较无耻, 一直想根据想象中的市场价格来收费,

FLOSS 们实在的多, 只想根据真实付出的劳动...

高下之分如此清晰, 那为什么闭源一直压 FLOSS 一头?

很简单, 闭源有专门资源针对当前市场习惯来编写营销故事...
